react/week1/todo/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+import ToDoItem from './ToDoItem';
+const toDoThing = [
+  { description: "get out of bed", deadline: "wed sep 13 2017" },
+  { description: "Brush Teeth", deadline: "Thu sep 13 2017" },
+  { description: "Eat breakfast", deadline: "wed sep 13 2017" },
+];
+function App() {
+  return (
+    <div className="App">
+   <ToDoList/>
+      <ul>
+          {toDoThing.map(todo => <ToDoItem description={todo.description} deadline={todo.deadline} />)}
+      </ul>
```

Comment:
I think this code here could potentially be converted to a Todos component. 

You would give this component a list of todos and it would handle the rest!

Reply:
Thanks for reviewing my hw..I changed some code accordingly your suggestions..